dist/access_check/MODULARITY.md

```diff
@@ -0,0 +1,44 @@
+# Rules for Modularity
```

Comment:
I like the contents in this md file. Do we consider moving it to the top directory level instead of nested here to gain more visibility? Plus the mention of relative path `src/` makes it more intuitive to have this file seated at the top level.

Reply:
It's currently experimental. I think we should do it when it becomes mandatory.

Reply:
Sounds good